test/propagated-properties.html

```diff
@@ -303,18 +311,30 @@
           }
         });
 
-        it('should have initial value for errorMessage property', () => {
+        it('should have initial value for errorMessage', () => {
           expect(dateTimePicker.errorMessage).to.equal('error-message');
           expect(customField.errorMessage).to.equal('error-message');
         });
 
-        it('should have initial value for required property', () => {
+        it('should have initial value for required', () => {
```

Comment:
This is for consistency with the other test descriptions (most of which also don't have the word "property").

Only in case of the `value` property I used "should have initial value for value property" (including "property") since having just "should have initial value for value" would have looked weird.